### PR TITLE
Treat 'Other' and 'No Program' codes differently

### PR DESCRIPTION
**Pull request by jesusaurus**
In order to only provide an 'Other' meta-code to Service Catalog while always adding a 'No Program' meta-code, configure the codes with different template parameters, and add a query-string parameter for including the 'Other' code.

### CODE REVIEW

File: mips_api/__init__.py [hunk 1]
```diff
@@ -148,9 +164,12 @@ def process_chart(chart_dict, omit_list, extra_dict):
...
 
-    # inject our extra programs at the beginning
-    for code, name in extra_dict.items():
-        out_chart[code] = name
+    # always inject "no program" code
+    out_chart[no_program] = "No Program"
+
+    # optionally inject "other" code
+    if _param_other_bool(params):
+        out_chart[other] = "Other"
```
Comment (zaro0508) [inline mips_api/__init__.py:172]: so you have the option to enable "Other" but you don't have the option to disable "No Program"?  These are non mips finance tags so it feels like these should just be treated the same.  Why not give the user the option to enable and disable both?

What if we just put these non-mips finance tags into a single list or dict and treat them the same?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: mips_api/__init__.py (hunk 1 above)
Comment (jesusaurus) [reply mips_api/__init__.py:172]: The problem with treating them the same is that the typical use case only includes one of the two values, but I suppose I can add a `disable_no_program_code` querystring parameter even though I don't see an immediate use case for it

Comment (zaro0508) [reply mips_api/__init__.py]: My concern is what happens if we have more non-mips codes?  I think adding another one of these should just be a configuration change however the way it's implemented we would need to add code.  With that being said I can accept this implementation if you feel that we will very likely never have another one non-mips code.

Comment (jesusaurus) [reply mips_api/__init__.py]: I'm pretty confident that we won't need any more extra codes, but if we do I can expand the lambda to accept two different lists of codes: one to include by default and that can be enabled with a parameter, but right now we just have one code of each type so the parsing to get a dict from a string seems unnecessary

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**